### PR TITLE
Update 08_FeedUpdates.md

### LINKED ISSUE #3634: [Feature] Automatic Refresh via Webhook
**Is your feature request related to a problem? Please describe.**
I use a shared hoster where i can not confiugre Cronjobs.
However i would like to automatically refresh my Subscriptions

**Describe the solution you'd like**
Call the ./app/actualize_script.php via HTTP

A direct call via HTTP get me to this error
FreshRSS error: This PHP script may only be invoked from command line!

### PR DESCRIPTION
**Pull request by marcohald**
Added hint to https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron

Closes https://github.com/FreshRSS/FreshRSS/issues/3634

Changes proposed in this pull request:

- Added Link to https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron

Pull request checklist:

- [x] documentation updated

### CODE REVIEW

**Comment by Frenzie** [comment]
It turns out @aledeg had more of a point than I thought, but a reference would surely make more sense in reverse?

**Comment by aledeg** [comment]
@marcohald  Thank you for your contribution :+1: 
Do you mind adding something to mention that there are online cron services that allows to work as a real cron? For instance, something like this service https://www.easycron.com/.

**Comment by aledeg** [comment]
> It turns out @aledeg had more of a point than I thought, but a reference would surely make more sense in reverse?

Maybe the reference can go both ways!

**Comment by Frenzie** [comment]
> Maybe the reference can go both ways!

I mean more like make that the one source of truth with maybe a short little summary and reference the detailed page. Doing it both ways is making things much worse. ;-)

File: docs/en/admins/08_FeedUpdates.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ FreshRSS is updated by the `./app/actualize_script.php` script. Knowing this, we
 
 **Note:** the following examples assume that FreshRSS is installed to `/usr/share/FreshRSS`. You'll need to modify the FreshRSS path to reflect your own system.
 
+**Note:** If you can't configure a Crojob look here for an alternative https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron
```
Comment (Alkarex) [inline docs/en/admins/08_FeedUpdates.md]:
```suggestion
**Note:** If you cannot configure a Cronjob, [look here for an alternative](../users/03_Main_view.md#online-cron).
```

Comment (Frenzie) [reply docs/en/admins/08_FeedUpdates.md]: I'd make that "an alternative using online cron" or something along those lines.

File: docs/en/admins/08_FeedUpdates.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ FreshRSS is updated by the `./app/actualize_script.php` script. Knowing this, we
 
 **Note:** the following examples assume that FreshRSS is installed to `/usr/share/FreshRSS`. You'll need to modify the FreshRSS path to reflect your own system.
 
+**Note:** If you cannot configure a Cronjob, [look here for an alternative](../users/03_Main_view.md#online-cron).
```
Comment (Alkarex) [inline docs/en/admins/08_FeedUpdates.md]:
```suggestion
**Note:** If you cannot configure a local Cronjob, [see an alternative using online cron](../users/03_Main_view.md#online-cron).
```

**Pull request merged**